RecoEgamma/EgammaIsolationAlgos/python/electronTrackIsolations_cfi.py

```diff
@@ -0,0 +1,92 @@
+import FWCore.ParameterSet.Config as cms
+
+trkIsol03CfgV1= cms.PSet(
```

Comment:
trkIsol03CfgV1 is not used in this PR.
Why is it needed?

Reply:
Two reasons
1) it was very useful in my testing to ensure that the new code with the same settings gave the same results as the old code
2) it allows people to easily switch back between the new and old variable for studies

None of this is super compelling I agree but I thought it was potentially useful enough given it costs extremely little to put there. But of course if you want, I can remove it.

Reply:
perhaps a comment could be added for every ID what they are supposed to be used for.
E.g. here "to reproduce legacy behavior" or smth like that

Reply:
agreed, already done for the next version.